server/db/schema.sql

```diff
@@ -0,0 +1,121 @@
+
+USE habitdb;
+
+
+-- ---
+-- Table 'users'
+-- 
+-- ---
+
+
+DROP TABLE IF EXISTS `users`;
+    
+CREATE TABLE `users` (
+  `id` INTEGER NOT NULL AUTO_INCREMENT DEFAULT NULL,
+  `username` VARCHAR(15) NOT NULL DEFAULT 'NULL',
```

Comment:
Very confusing default value here.  You may want to at least differentiate this default value from the `NULL` datatype using e.g. `'NO_USERNAME'`,  `'NULL_STRING'`, or the like.  Leaving as-is is liable to cause confusion later on.